migrations/versions/7bb5891a9f2e_lowercase_email.py

```diff
@@ -18,9 +18,20 @@
 
 
 def upgrade():
-    op.execute(" UPDATE users SET deleted_at = current_timestamp FROM (SELECT lower(_email) FROM users GROUP BY lower(_email) HAVING COUNT(*) > 1) as email where lower(_email) != _email;",
+    op.execute("UPDATE users SET deleted_at = current_timestamp FROM (SELECT lower(_email) FROM users GROUP BY lower(_email) HAVING COUNT(*) > 1) as email where lower(_email) != _email;",
+               execution_options=None)
+    op.execute("UPDATE users SET _email = concat(_email, '_') FROM (SELECT lower(_email) FROM users GROUP BY lower(_email) HAVING COUNT(*) > 1) as email where lower(_email) != _email;",
+               execution_options=None)
+    op.execute("update users set _email = lower(_email);",
+               execution_options=None)
+    op.execute("alter table users add check ( lower(_email) = _email );",
```

Comment:
No need to add check

Again, there is no index here, which makes this migration useless. Use citext extension to add a case insensitive column